docs/getting-started/dockstore-tools.rst

```diff
@@ -66,7 +66,7 @@ Installing the GitHub App is simple -- navigate to ``/my-tools``, click the ``+`
 
 Once you've installed our GitHub app on a repository or organization, you'll need to add a ``/.dockstore.yml`` file to
 the root directory of a branch of the repository that contains your tool. This file contains information like
-tool path, test parameter file, workflow name, etc. When a push is made on GitHub to a branch
+tool path, test parameter file, tool name, etc. When a push to a branch or tag is created on GitHub
```

Comment:
Seems to be a slight typo here, I think "When a push is made or a tag is created on GitHub"